packages/core/util/index.ts

```diff
@@ -659,7 +658,7 @@ export function makeAbortableReaction<T, U, V>(
           return undefined
         }
       },
-      async (data, mobxReactionHandle) => {
```

Comment:
unless the mobxReactionHandle is very useful to pass to the renderReaction, seems like we can remove it. avoids the typescript error